packages/measured-signalfx-reporter/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "measured-signalfx-reporter",
+  "description": "A Self Reporting Registry Reporter that knows how to report core metrics to SignalFx",
```

Comment:
nit: the description says "report" too many times.

How about: "A Registry Reporter that knows how to report core metrics to SignalFx"